### ISSUE TITLE
Profile language default on sign up?

### ISSUE THREAD

**Issue by peteruithoven** [loomio/loomio#1757]
Seeing https://github.com/loomio/loomio/pull/850 I expected my browser language preferences to be used as default for language. 
So I added dutch to the top of my browser (Firefox 33) languages and restarted my browser. Now when I check `window.navigator.language` in the Web developer Console, I get "nl" (dutch).
But when I create a new account on the [signup page](https://www.loomio.org/users/sign_up) it defaults to English.

**Comment by denjello** [comment]
Did you use the signup page in English?

On a side note, the Language switcher puts English always at the top of this list. I think that's unfair.

**Comment by peteruithoven** [comment]
I'll try to be more thorough. 
- Firefox's language is set to dutch (like described above)
- I didn't change the bottom language selector on the signup page, because I'm afraid most people won't see this option. Indeed it defaulted to English. 
The user profile was created with english as language. 

Attempt 2
- Firefox's language still set to dutch
- I didn't changed the bottom language selector to Nederlands (dutch). 
The user account was created with english as language. 
Still the user profile was created with english as language. 

So basically profile language is always set to English? No matter the language settings in a browser or the language selected on sign up.

**Comment by denjello** [comment]
I agree - the language switcher should hook on load and check that out.

**Comment by peteruithoven** [comment]
I opened a separate issue on the language selector: https://github.com/loomio/loomio/issues/1858
Maybe this issue could focus on the user profile default?

**Comment by robguthrie** [comment]
I'm gonna close this for now. The behaviour of the language stuff is like so:

We detect what language your browser tells us, and save it as user.detected_locale which we store and use when sending mail etc. If you want to use another locale you can select via the user preferences page, and that is saved as user.selected_locale which is always the used option if it is present.
